apps/content/schemaTypes/documents/hackathon.ts

```diff
@@ -0,0 +1,235 @@
+import {defineArrayMember, defineField, defineType} from 'sanity'
+import {RocketIcon, PlayIcon} from '@sanity/icons'
+
+export const hackathon = defineType({
+  name: 'hackathon',
+  type: 'document',
+  title: 'Hackathon',
+  icon: RocketIcon,
+  groups: [
+    {name: 'content', title: 'Content', default: true},
+    {name: 'seo', title: 'SEO & Publishing'},
+  ],
+  fields: [
+    defineField({
+      name: 'title',
+      type: 'string',
+      validation: (Rule) => Rule.required().error('Hackathon title is required'),
+      group: 'content',
+    }),
+    defineField({
+      name: 'slug',
+      type: 'slug',
+      description: 'URL-friendly identifier for this hackathon',
+      options: {
+        source: 'title',
+        maxLength: 96,
+      },
+      validation: (Rule) => Rule.required().error('Slug is required for URL generation'),
+      group: 'content',
+    }),
+    defineField({
+      name: 'pubDate',
+      type: 'datetime',
+      title: 'Publish Date',
+      description: 'When this hackathon announcement should be published',
+      options: {
+        timeStep: 30,
+      },
+      validation: (Rule) => Rule.required().error('Publish date is required'),
+      group: 'content',
+    }),
+    defineField({
+      name: 'deadline',
+      type: 'datetime',
+      title: 'Deadline',
+      description: 'When this hackathon submission deadline is',
+      options: {
+        timeStep: 30,
+      },
+      validation: (Rule) => Rule.required().error('Deadline is required'),
+      group: 'content',
+    }),
+    defineField({
+      name: 'description',
+      type: 'text',
+      description: 'Brief overview for previews and SEO',
+      validation: (Rule) => Rule.required().error('Description is required for SEO'),
+      group: 'content',
+    }),
+    defineField({
+      name: 'body',
+      type: 'markdown',
+      description: 'Full hackathon details, rules, and content',
+      validation: (Rule) => Rule.required().error('Body content is required'),
+      group: 'content',
+    }),
+    defineField({
+      name: 'episodes',
+      type: 'array',
+      title: 'Related Episodes',
+      description: 'Episodes associated with this hackathon',
+      of: [
+        defineArrayMember({
+          type: 'reference',
+          to: [{type: 'episode'}],
+          options: {
+            disableNew: true,
+          },
+        }),
+      ],
+      group: 'content',
+    }),
+    defineField({
+      name: 'submissionForm',
+      type: 'url',
+      title: 'Submission Form',
+      description: 'URL to the submission form for this hackathon',
+      validation: (Rule) =>
+        Rule.required()
+          .uri({scheme: ['http', 'https']})
+          .error('Submission form must be a valid URL'),
+      group: 'content',
+    }),
+    defineField({
+      name: 'rewards',
+      type: 'reference',
+      title: 'Rewards',
+      description: 'Rewards collection for this hackathon',
+      to: [{type: 'rewards'}],
```

Comment:
I think this will need to be the same a the FAQs. An "include as default" and query for that, but I need to be able to add one-off prizes based on sponsors, etc. without changing the list for previous hackathons and/or needing to duplicate the full list.

Let's move the `rewardItem` list here and filter on that default status for `initialValue`.